.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "can/cantools"]
 	path = common/cantools
 	url = ../cantools-autonomylab.git
+[submodule "ros/docker/gazebo/igvc_self_drive_sim"]
```

Comment:
I suggest we copy this in-tree rather than a submodule. It will be easier to make changes that way.